playwright.config.ts

```diff
@@ -18,6 +18,9 @@ const config: PlaywrightTestConfig = {
         ? undefined
         : {
               command: 'npm run dev',
+              env: {
+                  REACT_APP_DISABLE_CHECKS: 'true',
```

Comment:
disabled rtk-immutability checks for tests (in prod its disabled by default)
drastically improves stream tables performance